### PR TITLE
Frontend Modularization: Organizations

### PR DESCRIPTION
**Pull request by maryamkhidir**
### Feature or Bugfix

- Refactoring

### Details
- Organizations modularization into views, components and services
- Refactored relative imports used in the module to use absolute paths
- Removed unused scripts from services/graphql/Organizations
- Updated routes to reflect new file paths

### Relates
- https://github.com/awslabs/aws-dataall/issues/398

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/src/modules/Organizations/views/EnvironmentCreateForm.js
Comment (dlpzx) [inline frontend/src/modules/Organizations/views/EnvironmentCreateForm.js:1]: I understand why you placed this view inside the Organizations module: from Orgas you click on link environment and then the user gets redirected to this view. But the Environment creation form is more related with the Environment module. Even the API calls that it uses are imported from the Environment module

Comment (dlpzx) [reply frontend/src/modules/Organizations/views/EnvironmentCreateForm.js:1]: What do you think?

File: frontend/src/modules/Organizations/services/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export * from './archiveOrganization';
+export * from './createOrganization';
+export * from './getOrganization';
+export * from './inviteGroup';
+export * from './listOrganizationEnvironments';
```
Comment (dlpzx) [inline frontend/src/modules/Organizations/services/index.js:5]: I agree on placing the `listOrganizationEnvironments` here. It matches the decision in the backend

**Review by dlpzx — COMMENTED** [review]
looks good! I have only one comment on one of the views included in the Orga module

File: frontend/src/design/components/layout/DefaultSidebar.js [hunk 2]
```diff
@@ -82,7 +82,8 @@ export const DefaultSidebar = ({ openDrawer, onOpenDrawerChange }) => {
     const organizationsSection = {
       title: 'Organizations',
       path: '/console/organizations',
-      icon: <BiIcons.BiBuildings size={15} />
+      icon: <BiIcons.BiBuildings size={15} />,
+      active: config.modules.organizations.active
```
Comment (nikpodsh) [inline frontend/src/design/components/layout/DefaultSidebar.js:86]: As we discussed we don't want to disable organizations. So we can remove this line :)

Comment (maryamkhidir) [reply frontend/src/design/components/layout/DefaultSidebar.js:86]: Yes, I will update this.

File: frontend/src/modules/Organizations/views/EnvironmentCreateForm.js
Comment (maryamkhidir) [reply frontend/src/modules/Organizations/views/EnvironmentCreateForm.js:1]: Right, although the route points to organizations, as a module it is more related with environments.

File: frontend/src/design/components/layout/DefaultSidebar.js (hunk 2 above)
Comment (maryamkhidir) [reply frontend/src/design/components/layout/DefaultSidebar.js]: Updated with [9bb870c](https://github.com/awslabs/aws-dataall/pull/612/commits/9bb870c8763d34903d3e1901fc6f62c1bc69db65)

File: frontend/src/modules/Organizations/views/EnvironmentCreateForm.js
Comment (maryamkhidir) [reply frontend/src/modules/Organizations/views/EnvironmentCreateForm.js:1]: Updated with [9bb870c](https://github.com/awslabs/aws-dataall/pull/612/commits/9bb870c8763d34903d3e1901fc6f62c1bc69db65)

**Review by dlpzx — APPROVED** [review]
lgtm

**Pull request merged**